src/05-derivatives/01-introduction.md

```diff
@@ -28,6 +28,180 @@ in [Derived dataset and pipeline description][derived-dataset-description].
+that is placed under the datatype from which the model was derived.
+
+{{ MACROS___make_filetree_example(
+
+   {
+   "<pipeline_name>": {
+      "sub-<label>": {
+         "<datatype>": {
+            "model-<label>": {
+                "model_description.json": "",
+                "<model files>": ""
+            }
+         }
+      }
+   }
+   }
+
+) }}
+
+The specification of `<model files>` is [introduced below](#file-naming-conventions).
+
+### The `model_description.json` file
+
+Mirroring the special role of the `dataset_description.json` in any BIDS and BIDS Derivatives
+dataset, a `model_description.json` file MUST be found at the top level of every model folder
+(see template above).
+
+Example:
+
+```JSON
+{
+  "Name": "Diffusion Tensor Imaging (DTI)",
+  "FitMethod": "WLS",
+  "Implementation": "dipy.reconst.dti",
+  "ReferencesAndLinks": [
+      "https://doi.org/10.1016/S0006-3495(94)80775-1"
+  ]
+}
+```
+
+### Models file
+
+Template:
+
+{{ MACROS___make_filetree_example(
+
+   {
+   "<pipeline_name>": {
+      "sub-<label>": {
+         "<datatype>": {
+            "model-<label>": {
+                "model_description.json": "",
+                "<model files>": ""
+            },
+            "sub-<label>_models.tsv": "",
+            "sub-<label>_models.json": ""
+         }
+      }
+   }
+   }
+
+) }}
+
+A metadata file, `models.tsv` is OPTIONAL, accompanied by
+a `models.json` file that is REQUIRED, if and only if `models.tsv` is present.
+The purpose of the RECOMMENDED `models.tsv` file is to index the available
+`<model_id>` labels and describe properties of the models such as the datatype
+from which they are derived and a human-readable description of each model.
+
+If this file exists, it MUST contain the column `model_id`,
+which MUST consist of `model-<label>` values identifying one row for each model,
+followed by a list of optional columns describing models.
+Each model MUST be described by one and only one row.
+
+The RECOMMENDED `datatype` column SHOULD be one of the available
+[data type identifiers](../02-common-principles.md#definitions).
+
+The RECOMMENDED `description` column SHOULD be string containing a short description
+of the model. It is RECOMMENDED that the description is a single line of no more than
+50 words. It is also RECOMMENDED to avoid special characters (e.g., new lines, tabs).
+
+<!-- This block generates a columns table.
+The definitions of these fields can be found in
+  src/schema/objects/columns.yaml
+and a guide for using macros can be found at
+ https://github.com/bids-standard/bids-specification/blob/master/macros_doc.md
+-->
+{{ MACROS___make_columns_table(
+   {
+      "model_id": ("REQUIRED", "There MUST be exactly one row for each model."),
+      "datatype": "RECOMMENDED",
+      "description__models": "RECOMMENDED",
+   }
+) }}
+
+In this case, the `models.tsv` file could contain the following content:
```

Comment:
This example metadata file refers to a file/folder layout that is introduced further down, so this should probably also go further down.

Reply:
Resolved in https://github.com/bids-standard/bids-specification/pull/1280/commits/2e4e8ae8a4288048821419d3dc9e101a51b184db